src/components/Auth/ButtonSubmit/index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const ButtonSubmit = ({ label }) => {
+  return (
+    <div className="button-submit">
+      <button type="submit" className="btn btn--form-white">
```

Comment:
I think `btn--white` is enough, What do you think? And why do we need add `form` as prefix?

Reply:
I added the `form` prefix to declare it as a button meant to be used for form only. But I guess `btn--white` is enough. Changed the class name 🚀